app/src/main/java/com/alex/weatherapp/domain/useCase/GetTemperatureUseCase.kt

```diff
@@ -0,0 +1,11 @@
+package com.alex.weatherapp.domain.useCase
+
+import com.alex.weatherapp.domain.models.JsonDataClass
+import com.alex.weatherapp.domain.repository.RepositoryWeather
+
+class GetTemperatureUseCase(private val repositoryWeather: RepositoryWeather) {
+
+    suspend fun execute(): JsonDataClass? {
```

Comment:
```suggestion
    suspend operator fun invoke(): JsonDataClass? {
```

Can be simplified by using language feature "operator overriding"